src/selectors/rehydrate.js

```diff
@@ -1,14 +1,7 @@
 /* eslint-disable import/prefer-default-export */
```

Comment:
Without the other methods I think this might make more sense called something like ` src/selectors/forms.js`?